docs/sources/write/style-guide/inclusive-writing/index.md

```diff
@@ -1,79 +1,88 @@
 ---
-title: Inclusive writing
-description: Inclusive writing
-weight: 400
 aliases:
   - /docs/writers-toolkit/style-guide/inclusive-writing
   - /docs/writers-toolkit/write/style-guide/inclusive-writing
+date: 2024-05-16
+description: Understand the importantance of inclusive writing at Grafana Labs and learn how to write inclusively.
+weight: 400
 keywords:
   - inclusive writing
+title: Inclusive writing
 ---
 
 # Inclusive writing
 
-<!-- vale Grafana.GoogleWe = NO -->
+<!-- vale Grafana.We = NO -->
 <!-- This page talks about the voice and tone of our organization. -->
 
-When writing for Grafana Labs, always use inclusive terminology and phrasing. Avoid all statements that perpetuate gender, racial, or cultural stereotypes; do not use gendered words or demographically oriented terms that are irrelevant to the content.
+When writing for Grafana Labs, always use inclusive terminology and phrasing.
+Avoid all statements that perpetuate gender, racial, or cultural stereotypes;
+don't use gendered words or demographically oriented terms that are irrelevant to the content.
 
 ## Writing about people
 
 When writing about people, be compassionate, inclusive, and respectful.
 
-First and foremost, no matter the context, we always make sure to view people as people. When context and word count permits, we refer to them as people or people who maintain a particular area or discipline.
-
+No matter the context, we always make sure to view people as people.
 With that in mind, there are important distinctions between the ways that different people interact with our product and, thus, how we refer to those people.
 
 - Users are every person who uses our products, whether or not they use a commercial version of one of our products. Customers are people and companies who pay to use our products.
 - Visitors are people who visit our website for any reason, whether it's as a potential user, customer, or employee. We appreciate all interest in our hard work.
-- Our customers (and users) often have their own customers (and users). Generally speaking, we refer to these customers of our customers as users, because it does an okay job of encapsulating all the ways our various customers might interact with the people using their products.
+- Our customers (and users) often have their own customers (and users).
+  Generally speaking, we refer to these customers of our customers as users, because it does an okay job of encapsulating all the ways different customers might interact with the people using their products.
```

Comment:
```suggestion
  Generally speaking, we refer to these customers of our customers as users, because it does an OK job of encapsulating all the ways different customers might interact with the people using their products.
```